packages/suite-data/src/message-system/config/config.v1.json

```
@@ -147,10 +71,10 @@
             "conditions": [
                 {
                     "environment": {
-                        "desktop": "22.4.3",
+                        "desktop": "<=22.4.3",
```

This excludes the latest version, right? shouldn't it be the other way around?
```suggestion
                        "desktop": ">=22.4.3",
```

---

🤦